MediaAdmin/Reference/Strategies/MediaInNodeReferenceStrategy.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace OpenOrchestra\MediaAdmin\Reference\Strategies;
+
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
+use OpenOrchestra\Backoffice\Reference\Strategies\ReferenceStrategyInterface;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+
+/**
+ * Class MediaInNodeReferenceStrategy
+ */
+class MediaInNodeReferenceStrategy extends AbstractMediaReferenceStrategy implements ReferenceStrategyInterface
+{
+    /**
+     * @param mixed $entity
+     *
+     * @return boolean
+     */
+    public function support($entity)
+    {
+        return ($entity instanceof ReadNodeInterface);
+    }
+
+    /**
+     * @param mixed $entity
+     */
+    public function addReferencesToEntity($entity)
+    {
+        if ($this->support($entity)) {
```

Comment:
In others strategy + manager patterns, support is done in the manager loop, not directly in the strategies.